implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs

```diff
@@ -33,42 +34,33 @@ pub struct SecureChannelListenerNodeOpts {
 }
 
 impl CreateCommand {
-    pub fn run(self, options: CommandGlobalOpts) {
-        let cfg = options.config;
-        let node = extract_address_value(&self.node_opts.at).unwrap_or_else(|_| "".to_string());
-        let port = cfg.get_node_port(&node).unwrap();
-
-        connect_to(port, self, |ctx, cmd, rte| async {
-            create_listener(&ctx, cmd.address, cmd.authorized_identifier, rte).await?;
-            drop(ctx);
-            Ok(())
```

Comment:
I am not sure if the implementation in this PR correctly accounts for the `async` and  `drop` part

Reply:
Yeah, don't worry. Those parts are now handled internally by the `node_rpc` function. That's basically the reason behind this refactor.